dnload.py

```diff
@@ -1,8 +0,0 @@
-import os
```

Comment:
I understand your rationale on removing this python file, but I'd ask to retain it as it's not superfluous.

To elaborate, this tool is still for my personal use and used to create all the intros listed under faemiyah here (post-2013). I use it like in this example from the latest one:
https://github.com/faemiyah/faemiyah-demoscene_2018-08_40k-intro_cassini/blob/master/compile.sh

Different people on different platforms use the tool for development, and it's never 'ready' so it's never installed. The default use case is to clone this repo, and then clone the intro being worked on into a folder beside it. The python file existed so it'd be possible to easily refer to the file itself and it see its own basename.

Of course that same probe could now be done by locating the actual module and then using `-m ../dnload/dnload`, but then I'd have to test for __init__.py from inside that folder. All in all the python file made that probe easier. It was never meant to be used in any kind of install, since I never thought about any installation issues.

If there's no easy 'correct' solution for this, I'm all for changing the mode of operation in other projects using the script though.